test/integration/test_runner.py

```diff
@@ -26,6 +26,11 @@ def test_run_command(rc):
     status, exitcode = Runner(config=rc).run()
     assert status == 'successful'
     assert exitcode == 0
+    with open(os.path.join(rc.artifact_dir, 'command')) as f:
+        data = f.read()
```

Comment:
Seems easier to `json.load()` ?

Reply:
yeah, I wasn't confident that I'd be able to get a reasonable pattern out of the tmp directory